tests/testthat/test-renderer3-HitOrMissMonteCarlo.R

```diff
@@ -0,0 +1,171 @@
+  
+  viz.two <- ggplot()+
+    ggtitle("Monte Carlo Estimation of \u03C0")+
+    geom_tallrect(aes(
+      xmin=iter-0.5, xmax=iter+0.5),
+      clickSelects="iter",
+      alpha=0.5,
+      data=iteration)+
+    geom_hline(yintercept = pi, size=1)+
+    geom_path(aes(S.No, pi, key=iter),
+              showSelected="iter",
+              size=1,
+              data=data)+
+    geom_text(aes(iter, pi, key=iter, label=paste("\u03C0", "=", sprintf("%.6f", pi), sep = " ")),
+              showSelected=c("iter"),
+              size=20,
+              data=iteration)+
+    labs(x="Iterations", y="Value of \u03C0")+
+    scale_y_continuous(breaks = c(1, 2, 3, pi, 4, 5, 6, 7, 8))+
+    theme_bw()+
+    theme_animint(width=600, height=600)
+  
+  (viz.publish <- animint(viz.one,
+                          viz.two,
+                          title="Hit or Miss Monte Carlo Integration",
+                          time=list(variable="iter", ms=200),
+                          first=list(iter=1)))
+  return(viz.publish)
+}
+
+viz <- HitOrMissMonteCarlo()
+
+info <- animint2HTML(viz)
+
+# Test of needed data columns in TSV files
+chunk1.tsv <- file.path("animint-htmltest", "geom1_path_vizone_chunk1.tsv")
+chunk1 <- read.table(chunk1.tsv, sep="\t", header=TRUE,
+                     comment.char="", quote="")
+
+chunk2.tsv <- file.path("animint-htmltest", "geom2_point_vizone_chunk1.tsv")
+chunk2 <- read.table(chunk2.tsv, sep="\t", header=TRUE,
+                     comment.char="", quote="")
+
+chunk3.tsv <- file.path("animint-htmltest", "geom5_path_viztwo_chunk1.tsv")
+chunk3 <- read.table(chunk3.tsv, sep="\t", header=TRUE,
+                     comment.char="", quote="")
+
+chunk4.tsv <- file.path("animint-htmltest", "geom6_text_viztwo_chunk1.tsv")
+chunk4 <- read.table(chunk4.tsv, sep="\t", header=TRUE,
+                     comment.char="", quote="")
+
+chunk5.tsv <- file.path("animint-htmltest", "geom4_hline_viztwo_chunk1.tsv")
+chunk5 <- read.table(chunk5.tsv, sep="\t", header=TRUE,
+                     comment.char="", quote="")
+
+chunk6.tsv <- file.path("animint-htmltest", "geom3_tallrect_viztwo_chunk1.tsv")
+chunk6 <- read.table(chunk6.tsv, sep="\t", header=TRUE,
+                     comment.char="", quote="")
+
+test_that("Integration Curve data contains expected columns", {
+  path_expected_cols <- c("x", "y", "group")
+  point_expected_cols <- c("shape", "colour", "x", "y", "key", "showSelectedlegendshapecolour", "showSelected1", "fill")
+  
+  expect_identical(sort(names(chunk1)), sort(path_expected_cols))
+  expect_identical(sort(names(chunk2)), sort(point_expected_cols))
+})
+
+test_that("Pi estimation graph data contains expected columns", {
+  path_expected_cols <- c("x", "y", "showSelected1", "key", "group")
+  text_expected_cols <- c("x", "y", "key", "label", "showSelected1", "group")
+  tallrect_expected_cols <- c("xmin", "xmax", "clickSelects", "group")
+  hline_expected_cols <- c("yintercept")
+  
```

Comment:
these expected columns tests are not appropriate, because the names/ordering of columns in csv files is a compiler implementation detail, and the test-renderer* files should test visual/renderer functionality. please remove.

Reply:
Okay got it.
Let me remove this in the next commit.

Reply:
I've removed these checks.